src/lv_core/lv_obj.h

```diff
@@ -169,6 +169,13 @@ typedef struct
 } lv_reailgn_t;
 #endif
 
+enum {
+	LV_DRAG_DIR_HOR = 0x1,
+	LV_DRAG_DIR_VER = 0x2
+};
+
+typedef uint8_t lv_drag_direction_t;
```

Comment:
IMO `lv_drag_dir_t` is also clear and shorter. In addition for (micro)python binding, we should keep the enum names and elements consistent.